core/domain/services/custom_post_types/RegisterCustomPostTypes.php

```diff
@@ -168,38 +168,112 @@ private function getLabels($singular_name, $plural_name, $singular_slug, $plural
             'singular_slug'      => $singular_slug,
             'plural_slug'        => $plural_slug,
             'add_new'            => sprintf(
+                /* Translators: Post Type Label */
                 esc_html_x('Add %s', 'Add Event', 'event_espresso'),
                 $singular_name
             ),
             'add_new_item'       => sprintf(
+                /* Translators: Post Type Label */
                 esc_html_x('Add New %s', 'Add New Event', 'event_espresso'),
                 $singular_name
             ),
             'edit_item'          => sprintf(
+                /* Translators: Post Type Label */
                 esc_html_x('Edit %s', 'Edit Event', 'event_espresso'),
                 $singular_name
             ),
             'new_item'           => sprintf(
+                /* Translators: Post Type Label */
                 esc_html_x('New %s', 'New Event', 'event_espresso'),
                 $singular_name
             ),
             'all_items'          => sprintf(
+                /* Translators: Post Type Label */
                 esc_html_x('All %s', 'All Events', 'event_espresso'),
                 $plural_name
             ),
             'view_item'          => sprintf(
+                /* Translators: Post Type Label */
                 esc_html_x('View %s', 'View Event', 'event_espresso'),
                 $singular_name
             ),
+            'view_items' => sprintf(
```

Comment:
Spacing is different here than the above items?
(phpcs didn't highlight this so just checking)

Reply:
That's just an personal preference, not necessary.